modules/client/src/main/kotlin/jp/co/soramitsu/iroha2/Iroha2Client.kt

```diff
@@ -79,14 +82,36 @@ open class Iroha2Client(
             .status.value
     }
 
+    /**
+     * Sends configuration request
+     */
+    suspend inline fun <reified T> configuration(
```

Comment:
This is not type safe. Not clear what `T` actually can be only a map or a string. Library's user not necessary knows about types.  I would separate this endpoint to two endpoints - one for configuration, and another for docs. This way returned value's type will be certain